service/test/lib/integration/teamOpsTest.js

```diff
@@ -466,14 +474,14 @@ lab.experiment('TeamOps', () => {
     teamOps.readTeam({ id: 1, organizationId: 'WONKA' }, (err, team) => {
       expect(err).to.not.exist()
       expect(team).to.exist()
-      expect(team.policies).to.equal([{ id: 1, name: 'Director', version: '0.1' }])
+      expect(team.policies).to.equal([{ id: 'policyId1', name: 'Director', version: '0.1' }])
 
-      teamOps.deleteTeamPolicy({ teamId: 1, policyId: 1, organizationId: 'WONKA' }, (err, team) => {
+      teamOps.deleteTeamPolicy({ teamId: 1, policyId: 'policyId1', organizationId: 'WONKA' }, (err, team) => {
```

Comment:
I see a lot of this occurrences and I wonder if I didn't fix them in my PR or if this is a side-effect of the rebase